specification/specs/common.ts

```diff
@@ -103,8 +103,13 @@ class DateField {
   include_unmapped?: boolean;
 }
 
+/** Rather then documenting `object` this specifies places in the response that depend on the mapping and or request
+ * e.g the result of a script can be anything from value types to objects  */
 class UserDefinedValue {}
 
+/** Documents a place that can return ANY document from ANY index that can not be directly related back to a generic */
+class LazyDocument {}
```

Comment:
Should we use this one for `MultiGet` as well?
Currently isn't a problem, as the `MultiGet` generic defaults to `unknown`.

Reply:
we should :)
https://github.com/elastic/elastic-client-generator/issues/71